Sources/Misc/StoreKitVersion.swift

```diff
@@ -35,6 +35,13 @@ public extension StoreKitVersion {
     /// Let RevenueCat use the most appropiate version of StoreKit
     static let `default` = Self.storeKit2
 
+    #if ENABLE_CUSTOM_ENTITLEMENT_COMPUTATION
+
+    /// Let RevenueCat use the most appropiate version of StoreKit
+    static let defaultForCustomEntitlementComputation = Self.storeKit1
```

Comment:
I wonder, would it make more sense to have it like this?

```suggestion
    #if ENABLE_CUSTOM_ENTITLEMENT_COMPUTATION
  
    /// Let RevenueCat use the most appropiate version of StoreKit
    static let `default` = Self.storeKit1
    
    #else

    /// Let RevenueCat use the most appropiate version of StoreKit
    static let default = Self.storeKit2
    
    #endif
```

Reply:
Hmm right, that probably makes more sense 😅. Thanks!